CONTRIBUTING.md

```diff
@@ -0,0 +1,320 @@
+- [Contribute to `algieba`](#contribute-to-algieba)
```

Comment:
Do we actually need this? :/

Reply:
no we don't need that specific contribution guide, but since it's in `zosma` (and i presume `torch`) we should at least be consistent

Reply:
I recall we decided to remove it; which is why it is removed from the initial PR of torch.